<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>File Encryptor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f5f5f5;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="centralLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane { border: 1px solid #c0c0c0; background: #ffffff; }
                      QTabBar::tab { padding: 8px 16px; }
                      QTabBar::tab:selected { background: #e0e0e0; }</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Encrypt</string>
       </attribute>
       <layout class="QVBoxLayout" name="encryptLayout">
        <item>
         <layout class="QHBoxLayout" name="encryptButtonLayout">
          <item>
           <widget class="QLabel" name="statusLabel">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #333333;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="encryptButtonSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="addFilesButton">
            <property name="text">
             <string>Add Files</string>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 6px; background-color: #0078d4; color: white; border-radius: 4px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeButton">
            <property name="text">
             <string>Remove</string>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 6px; background-color: #d83b01; color: white; border-radius: 4px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearButton">
            <property name="text">
             <string>Clear</string>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 6px; background-color: #d83b01; color: white; border-radius: 4px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="encryptButton">
            <property name="text">
             <string>Encrypt</string>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 6px; background-color: #2e7d32; color: white; border-radius: 4px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="fileList">
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid #c0c0c0; background: white; color: #333333; padding: 5px;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Decrypt</string>
       </attribute>
       <layout class="QVBoxLayout" name="decryptLayout">
        <item>
         <widget class="QLabel" name="fileLabel">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Encrypted File:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="fileInputLayout">
          <item>
           <widget class="QLineEdit" name="encryptedFilePath">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="placeholderText">
             <string>Select .axine file</string>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 6px; border: 1px solid #c0c0c0; border-radius: 4px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="browseButton">
            <property name="text">
             <string>Browse</string>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 6px; background-color: #0078d4; color: white; border-radius: 4px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="keyLabel">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Key (16 hex chars):</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="keyInputLayout">
          <item>
           <widget class="QLineEdit" name="keyInput">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="placeholderText">
             <string>Enter key (e.g., 0123456789ABCDEF)</string>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 6px; border: 1px solid #c0c0c0; border-radius: 4px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="browseKeyButton">
            <property name="text">
             <string>Browse Key</string>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 6px; background-color: #0078d4; color: white; border-radius: 4px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="decryptButtonLayout">
          <item>
           <spacer name="decryptSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="decryptButton">
            <property name="text">
             <string>Decrypt</string>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 6px; background-color: #2e7d32; color: white; border-radius: 4px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="decryptStatusLabel">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #333333;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>